app/src/renderer/components/common/NiSessionImport.vue

```diff
@@ -38,6 +38,13 @@
         placeholder="Must be exactly 12 words")
       form-msg(name='Seed' type='required' v-if='!$v.fields.importSeed.required')
 
+    form-group(field-id="error-collection" field-label=' ')
+      .ni-field-checkbox
+        .ni-field-checkbox-input
+          input#sign-up-warning(type="checkbox" v-model="fields.errorCollection")
+        label.ni-field-checkbox-label(for="error-collection")
+          | I'd like to opt in for remote error tracking to help improve Voyager
```

Comment:
i'm sorry this is annoying - but there should be a period at the end of this sentence.